xsl/xccdf-resources/openscap.js

```diff
@@ -108,12 +109,152 @@ function ruleSearch()
         $("#search-matches").html("No rules match your search criteria!");
 }
 
+var is_original = true;
+var original_treetable = null;
+
 $(document).ready( function() {
     $("#result-details").hide();
     $(".js-only").show();
+	$(".form-group select").val("default");
     $(".toggle-rule-display").each(function(){
         toggleRuleDisplay(this);
     });
-
-    $(".treetable").treetable({ column: 0, expandable: true, initialState : "expanded", indent : 0 });
+    original_treetable = $(".treetable").clone();
+    $(".treetable").treetable({ column: 0, expandable: true, clickableNodeNames: true, initialState : "expanded", indent : 0 });
+    is_original = true;
+	stripeTreeTable();
 });
+
+function resetTreetable() {
```

Comment:
Is this function used / usable outside of group-by functionality? If not then why doesn't the name signify that?